scripts/preload_models.py

```diff
@@ -10,28 +10,31 @@
 import transformers
 import os
 import warnings
+import torch
 import urllib.request
+import zipfile
+import traceback
 
 transformers.logging.set_verbosity_error()
 
 # this will preload the Bert tokenizer fles
-print('preloading bert tokenizer...', end='')
-
-tokenizer = BertTokenizerFast.from_pretrained('bert-base-uncased')
+print('Loading bert tokenizer (ignore deprecation errors)...', end='')
+with warnings.catch_warnings():
+    warnings.filterwarnings('ignore', category=DeprecationWarning)
+    tokenizer = BertTokenizerFast.from_pretrained('bert-base-uncased')
```

Comment:
Am I crazy, or are we loading BERT into `tokenizer`, overwriting `tokenizer` with `CLIPTokenizer.from_pretrained()` below, and never referring to BERT again?

Reply:
I'm referring to current 'main', btw ([ref](https://github.com/invoke-ai/InvokeAI/blob/e5ab07091d50f7f6949bb9487f843188575dbf23/scripts/preload_models.py#L35)); this wasn't added in *this* PR.

Reply:
I believe that the BERT tokenizer is used by the embodding manager. At least there are a lot of references to it in the embedding_manager.py and encoders/modules.py files.

Reply:
Right, but the BERT version of the `tokenizer` is permanently overwritten by the CLIP version, so **nothing** is "using" it currently. 

Relevant snippet (note that there's no conditional, nor is BERT "handed off" to another alias of some kind):

```python
# this will preload the Bert tokenizer fles
print('preloading bert tokenizer...', end='')

tokenizer = BertTokenizerFast.from_pretrained('bert-base-uncased')
print('...success')

...
*snip 7 irrelevant lines*
...

print('preloading CLIP model...',end='')
sys.stdout.flush()

tokenizer = CLIPTokenizer.from_pretrained(version)
```